npm run build doesn't work on Windows_NT 6.3.9600

%npm run build

> swagger-ui@2.2.3 build c:\emacs\mova\swagger-ui
> npm run handlebars && gulp


> swagger-ui@2.2.3 handlebars c:\emacs\mova\swagger-ui
> ./node_modules/handlebars/bin/handlebars ./src/main/template -f ./src/main/template/templates.js && gulp handlebars

'.' n o   reconhecido como um comando interno
ou externo, um programa oper vel ou um arquivo em lotes.

(That's saying . is not a known command.  That's a message from Windows' command prompt.  It's written in Portuguese with the wrong encoding --- as usual.)

npm ERR! Windows_NT 6.3.9600
npm ERR! argv "c:\\emacs\\nodejs\\node.exe" "c:\\emacs\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "handlebars"
npm ERR! node v4.5.0
npm ERR! npm  v2.15.9
npm ERR! code ELIFECYCLE
npm ERR! swagger-ui@2.2.3 handlebars: `./node_modules/handlebars/bin/handlebars ./src/main/template -f ./src/main/template/templates.js && gulp handlebars`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the swagger-ui@2.2.3 handlebars script './node_modules/handlebars/bin/handlebars ./src/main/template -f ./src/main/template/templates.js && gulp handlebars'.
npm ERR! This is most likely a problem with the swagger-ui package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./node_modules/handlebars/bin/handlebars ./src/main/template -f ./src/main/template/templates.js && gulp handlebars
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs swagger-ui
npm ERR! Or if that isn't available, you can get their info via:
npm ERR! 
npm ERR!     npm owner ls swagger-ui
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     c:\emacs\mova\swagger-ui\npm-debug.log

npm ERR! Windows_NT 6.3.9600
npm ERR! argv "c:\\emacs\\nodejs\\node.exe" "c:\\emacs\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build"
npm ERR! node v4.5.0
npm ERR! npm  v2.15.9
npm ERR! code ELIFECYCLE
npm ERR! swagger-ui@2.2.3 build: `npm run handlebars && gulp`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the swagger-ui@2.2.3 build script 'npm run handlebars && gulp'.
npm ERR! This is most likely a problem with the swagger-ui package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run handlebars && gulp
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs swagger-ui
npm ERR! Or if that isn't available, you can get their info via:
npm ERR! 
npm ERR!     npm owner ls swagger-ui
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     c:\emacs\mova\swagger-ui\npm-debug.log
%

%pwd
c:/emacs/mova/swagger-ui

%cat npm-debug.log
/usr/bin/cat: npm-debug.log: No such file or directory
%

--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
   "main": "dist/swagger-ui.js",
   "scripts": {
     "build": "npm run handlebars && gulp",
-    "handlebars": "./node_modules/handlebars/bin/handlebars ./src/main/template -f ./src/main/template/templates.js && gulp handlebars",
+    "handlebars": "handlebars ./src/main/template -f ./src/main/template/templates.js && gulp handlebars",
     "serve": "gulp serve",
     "prejshint": "gulp",
     "jshint": "jshint .",

